CogniteSdk.Types/DataPoints/DataPointsCreate.cs

```diff
@@ -4,17 +4,17 @@
 using CogniteSdk.Types.Common;
 using System.Collections.Generic;
 
-namespace CogniteSdk.DataPoints
+namespace CogniteSdk
 {
     /// <summary>
     /// Data Points write type
     /// </summary>
-    public class DataPointsWriteDto
+    public class DataPointsCreate
```

Comment:
This class doesn't seem to be used for anything.  Datapoints are written instead through NumericDatapoint/DataPointInsertionItem/DataPointInsertionRequest.

Reply:
Good catch! Probably a left-over from the time we used JSON to write datapoints. But now it uses Protobuf instead which is much more efficient when writing a high number of data-points. We can probably just remove this class